src/Product/SearchProvider.php

```diff
@@ -456,7 +456,7 @@ private function hideUselessFacets(array $facets, $totalProducts)
                 (
                     count($facet->getFilters()) === 1
                     && $totalFacetProducts < $totalProducts
-                    && $usefulFiltersCount > 1
+                    && $usefulFiltersCount > 0
```

Comment:
This can lead to massive changes in the behavior, sure about it ?

Reply:
![image](https://user-images.githubusercontent.com/1462701/64518083-d42a3d00-d2f1-11e9-9276-b1866a3ca0ea.png)
As you can see in this image, you have only one `Color` which is not `Not available`, without the change, it doesn't display it.
More, because the facet doesn't have the same number of products than the total, it means the filter is useful and can be used.